hal/network/ncp_client/quectel/quectel_ncp_client.cpp

```diff
@@ -964,10 +964,6 @@ int QuectelNcpClient::initReady(ModemState state) {
         // Select either internal or external SIM card slot depending on the configuration
         CHECK(selectSimCard());
 
-        // Just in case disconnect
-        int r = CHECK_PARSER(parser_.execCommand("AT+COPS=2"));
-        // CHECK_TRUE(r == AtResponse::OK, SYSTEM_ERROR_UNKNOWN);
-
```

Comment:
Example for longer reg time with and without using `COPS=2` on `EG91`:

- With `COPS=2`

```
0000016015 [ncp.at] TRACE: > AT+CGDCONT=1,"IP","super"
0000016047 [ncp.at] TRACE: < OK
0000016048 [ncp.at] TRACE: > AT+CFUN=1,0
0000016051 [ncp.at] TRACE: < OK
0000016052 [ncp.at] TRACE: > AT+CREG=2
0000016054 [ncp.at] TRACE: < OK
0000016055 [ncp.at] TRACE: > AT+CGREG=2
0000016059 [ncp.at] TRACE: < OK
0000016060 [ncp.at] TRACE: > AT+CEREG=2
0000016065 [ncp.at] TRACE: < OK
0000016066 [ncp.client] TRACE: NCP connection state changed: 1
0000016076 [net.pppncp] TRACE: NCP event 2
0000016077 [net.pppncp] TRACE: State changed event: 1
0000016082 [ncp.at] TRACE: > AT+COPS?
0000016088 [ncp.at] TRACE: < +COPS: 2
0000016088 [ncp.at] TRACE: < OK
0000016093 [ncp.at] TRACE: > AT+COPS=0,2
0000016082 [net.ppp.client] TRACE: PPP thread event LOWER_DOWN
0000016111 [ncp.at] TRACE: < OK
0000016112 [ncp.at] TRACE: > AT+CREG?
0000016115 [ncp.at] TRACE: < +QIND: SMS DONE
0000016116 [ncp.at] TRACE: < +CREG: 2,0
0000016117 [ncp.at] TRACE: < OK
0000016124 [ncp.at] TRACE: > AT+CGREG?
0000016127 [ncp.at] TRACE: < +CGREG: 2,0
0000016128 [ncp.at] TRACE: < OK
0000016129 [ncp.at] TRACE: > AT+CEREG?
0000016135 [ncp.at] TRACE: < +CEREG: 2,0
0000016136 [ncp.at] TRACE: < OK
0000031140 [ncp.at] TRACE: > AT+CEER
0000031145 [ncp.at] TRACE: < +CEER: 0,-1
0000031146 [ncp.at] TRACE: < OK
0000031147 [ncp.at] TRACE: > AT+CREG?
0000031149 [ncp.at] TRACE: < +CREG: 2,2
0000031152 [ncp.at] TRACE: < OK
0000031153 [ncp.at] TRACE: > AT+CGREG?
0000031156 [ncp.at] TRACE: < +CGREG: 2,2
0000031164 [ncp.at] TRACE: < OK
0000031165 [ncp.at] TRACE: > AT+CEREG?
0000031168 [ncp.at] TRACE: < +CEREG: 2,2
0000031175 [ncp.at] TRACE: < OK
0000046175 [ncp.at] TRACE: > AT+CEER
0000046178 [ncp.at] TRACE: < +CEER: 0,-1
0000046179 [ncp.at] TRACE: < OK
0000046180 [ncp.at] TRACE: > AT+CREG?
0000046182 [ncp.at] TRACE: < +CREG: 2,2
0000046183 [ncp.at] TRACE: < OK
0000046189 [ncp.at] TRACE: > AT+CGREG?
0000046192 [ncp.at] TRACE: < +CGREG: 2,2
0000046194 [ncp.at] TRACE: < OK
0000046200 [ncp.at] TRACE: > AT+CEREG?
0000046203 [ncp.at] TRACE: < +CEREG: 2,2
0000046205 [ncp.at] TRACE: < OK
0000054205 [ncp.at] TRACE: < +CREG: 5,"6A41","7F3CC11",7
0000054306 [ncp.at] TRACE: < +CGREG: 5,"6A41","7F3CC11",7
```

- Without `COPS=2` (shorter!)

```
0000067363 [ncp.at] TRACE: > AT+CGDCONT=1,"IP","super"
0000067393 [ncp.at] TRACE: < OK
0000067394 [ncp.at] TRACE: > AT+CFUN=1,0
0000067396 [ncp.at] TRACE: < OK
0000067397 [ncp.at] TRACE: > AT+CREG=2
0000067399 [ncp.at] TRACE: < OK
0000067400 [ncp.at] TRACE: > AT+CGREG=2
0000067407 [ncp.at] TRACE: < OK
0000067408 [ncp.at] TRACE: > AT+CEREG=2
0000067411 [ncp.at] TRACE: < OK
0000067411 [ncp.client] TRACE: NCP connection state changed: 1
0000067419 [net.pppncp] TRACE: NCP event 2
0000067422 [net.pppncp] TRACE: State changed event: 1
0000067430 [net.ppp.client] TRACE: PPP thread event LOWER_DOWN
0000067431 [ncp.at] TRACE: > AT+COPS?
0000067441 [ncp.at] TRACE: < +COPS: 0,0,"AT&T Twilio",7
0000067442 [ncp.at] TRACE: < OK
0000067445 [ncp.at] TRACE: > AT+CREG?
0000067452 [ncp.at] TRACE: < +CREG: 2,5,"6A41","7F46703",7
0000067453 [ncp.at] TRACE: < OK
0000067456 [ncp.at] TRACE: > AT+CGREG?
0000067463 [ncp.at] TRACE: < +CGREG: 2,5,"6A41","7F46703",7
0000067465 [ncp.at] TRACE: < OK
0000067467 [ncp.at] TRACE: > AT+CEREG?
0000067474 [ncp.at] TRACE: < +CEREG: 2,5,"6A41","7F46703",7
0000067475 [ncp.at] TRACE: < OK
```